model/event/BatteryBeforeRemoveEvent.php

```diff
@@ -19,6 +19,29 @@
 
 namespace oat\taoBattery\model\event;
 
-class BatteryBeforeRemoveEvent extends AbstractBatteryEvent
+class BatteryBeforeRemoveEvent
```

Comment:
Why did you removed `extends AbstractBatteryEvent`?

Reply:
Because this is not implementing my interface. It's used only in rework assignment branch and it's not included in my work.